components/script/script_thread.rs

```diff
@@ -1217,7 +1217,8 @@ impl ScriptThread {
 
         if let Some(root_context) = self.browsing_context.get() {
             for it_context in root_context.iter() {
-                let current_url = it_context.active_document().url().to_string();
+                let doc = it_context.active_document();
+                let current_url = doc.url().to_string();
```

Comment:
This looks like a premature optimization - no code is using the `doc` variable at all, except for `current_url`, so we can just keep the previous code here (i.e. everything on 1 line).